com.thincast.client.appdata.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally appdata files should be part of client/webaccess tarballs as this way you can actually maintain them as part of software release cycle but if you think it fits better here, ignore this comment.

Reply:
@barthalion will do that for the next version. Need to wait for a new version to be build (that might take a few days until there are enough fixes)